### PR TITLE
Generate yaml / python data files for vert_interp

### PR DESCRIPTION
**Pull request by brigaudet**
Generate yaml / python data files for vert_interp subroutine in utils/tracer_data.F90 (one vertical column).

### CODE REVIEW

File: components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90 [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+        !start by adding an input string
+        call write_input_output_header(unit_input, unit_output,yaml%lchnk_print,yaml%col_print, &
+             'vert_interp',yaml%nstep_print, yaml%lev_print)
+
+        ! add code for writing data here
+        
+        call write_var(unit_input,unit_output,'ncol',ncol)
+        call write_var(unit_input,unit_output,'levsiz',levsiz)
+        call write_var(unit_input,unit_output,'pin',pin(yaml%col_print,:))
```
Comment (odiazib) [inline components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90]: @brigaudet Could you please generate files for all columns associated with this MPI rank?

e.i.,  
`call write_var(unit_input,unit_output,'pin',pin(yaml%col_print,:))` ->

`call write_var(unit_input,unit_output,'pin',pin(:,:))`

**Review by odiazib — APPROVED** [review]
It looks great. Only one comment for additional data.

File: components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90 (hunk 1 above)
Comment (brigaudet) [reply components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90]: How does this work?

Comment (odiazib) [reply components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90]: I meant to say, as an example, replace `call write_var(unit_input, unit_output, 'pin', pin(yaml%col_print, :))` with `call write_var(unit_input, unit_output, pin, pin(:, :))`.

Comment (brigaudet) [reply components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90]: Can I go ahead and merge this pull request (along with all the other outstanding ones)?

Comment (odiazib) [reply components/eam/src/chemistry/yaml/tracer_data/f90_yaml/vert_interp_beg_yml.f90]: Yes.

**Comment by brigaudet** [comment]
image

All tests passed, will merge.

**Pull request merged**